packages/sdk-rtl/src/nodeSession.ts

```diff
@@ -33,26 +33,19 @@ import {
 } from './transport'
 import { AuthToken } from './authToken'
 import { NodeTransport } from './nodeTransport'
-import {
-  IApiSettings,
-  strLookerClientId,
-  strLookerClientSecret,
-} from './apiSettings'
+import { IApiSettings } from './apiSettings'
 import { AuthSession, IAccessToken, IError } from './authSession'
-import { defaultApiVersion } from './constants'
-import { getenv } from './nodeSettings'
 
 const strPost: HttpMethod = 'POST'
 const strDelete: HttpMethod = 'DELETE'
 
 export class NodeSession extends AuthSession {
-  private readonly apiPath: string = ''
+  private readonly apiPath: string = '/api/3.1'
```

Comment:
Even though you've removed this from the RTL constants, seems like you would still want it to be configurable when being initialized as part of a particular SDK.

Reply:
Agreed. I was deferring that until 3.1 is deprecated or we need something in a more recent API version. Can certainly come back to this.